packages/dataviews/src/field-types/datetime.tsx

```diff
@@ -40,7 +39,15 @@ export default {
 	},
 	enableSorting: true,
 	filterBy: {
-		defaultOperators: [ OPERATOR_IS, OPERATOR_IS_NOT ],
-		validOperators: [ OPERATOR_IS, OPERATOR_IS_NOT ],
+		defaultOperators: [
+			OPERATOR_IS,
+			OPERATOR_IS_NOT,
+			OPERATOR_BETWEEN,
+		],
+		validOperators: [
+			OPERATOR_IS,
+			OPERATOR_IS_NOT,
+			OPERATOR_BETWEEN,
```

Comment:
How does the `between` operator works for datetime type fields (see in `yarn workspace @automattic/dataviews storybook:start`)?

https://github.com/user-attachments/assets/aaa36355-b9ef-4b85-89f6-6f9c5f203165

Reply:
I just noticed this in the PR description:

> While this functionality is also compatible with datetime fields, the UI for date selection has not yet been implemented, so it is not enabled by default for date fields.

It's okay to only add support for integers. We shouldn't enable this operator until it works for datetime fields.

Reply:
Right, we should disable it for date fields until it works. Reverted changes in 1dbcf860c0a8cde5448f4e0c9cc23100387faa57.